addons/armaos/functions/fnc_terminal_updateBufferVisable.sqf

```diff
@@ -0,0 +1,32 @@
+params ["_computer"];
+
+private _terminal = _computer getVariable "AE3_terminal";
+
+private _terminalBuffer = _terminal get "AE3_terminalBuffer";
+private _terminalMaxRows = _terminal get "AE3_terminalMaxRows";
+
+private _terminalBufferLength = count _terminalBuffer;
+private _lastBufferLineIndex = _terminalBufferLength - 1;
+
+// + to preserve reference and force copy
+private _terminalBufferVisable = +_terminalBuffer;
+
+//hint format ["BUFFER: \n ***%1*** \n\n BUFFER VISABLE: \n ***%2***", _terminalBuffer, _terminalBufferVisable];
```

Comment:
Dead code

Reply:
Removed in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/bdeec0439c3a89ef1583c3b663196a37a6c51e0b